### PR TITLE
Add debugging bad certificates to flotta-dev-cli

### PR DESCRIPTION
**Pull request by arielireni**
Added documentation on updating the certificates under troubleshooting -> debug the device section.

Signed-off-by: arielireni <aireni@redhat.com>

### CODE REVIEW

File: _posts/2022-07-20-developer-cli.markdown [hunk 1]
```diff
@@ -106,6 +106,16 @@ As things might not always be clear, here are some tips to help you:
 * Workloads are run under the _flotta_ user. Workloads debugging needs to be done also as _flotta_ user on the device.
 
 ### Debug the device
+For registering a device, the operator should download certificates and set them the correct permissions:
+```shell
+# remove old certs, if exists
+$ sudo rm /tmp/*.pem
+# download new certs
+$ make get-certs
```
Comment (masayag) [inline _posts/2022-07-20-developer-cli.markdown:114]: pls specify that this command should be executed from the flotta-operator project

File: _posts/2022-07-20-developer-cli.markdown [hunk 2]
```diff
@@ -106,6 +106,16 @@ As things might not always be clear, here are some tips to help you:
 * Workloads are run under the _flotta_ user. Workloads debugging needs to be done also as _flotta_ user on the device.
 
 ### Debug the device
+For registering a device, the operator should download certificates and set them the correct permissions:
```
Comment (masayag) [inline _posts/2022-07-20-developer-cli.markdown:109]: "the operator" - isn't precise. it is the flotta-edge-api that creates the certificates.

"I'd phrase this as: "In order to register the device, the Flotta Edge API certificates need to be used to establish communication. To download the certificates run the following from flotta-operator project."

Comment (arielireni) [reply _posts/2022-07-20-developer-cli.markdown]: done

File: _posts/2022-07-20-developer-cli.markdown (hunk 1 above)
Comment (arielireni) [reply _posts/2022-07-20-developer-cli.markdown:114]: replaced `# download new certs` with `# download new certs, from the flotta-operator project`

**Review by masayag — APPROVED** [review]

**Pull request merged**